File: 231113/특정 숫자 맞추기/catching-specific-number.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);

        while (true) {
            int n = sc.nextInt();

            if (n != 25) {
                System.out.println(n < 25 ? "Higher" : "Lower");
            }
            else {
                System.out.println("Good");
                break;
            }
        }
    }
}

File: trail0/특정 수 맞히기/catching-specific-number.py
import sys

input = sys.stdin.readline
print = sys.stdout.write

while True:
    n = int(input())

    if n > 25:
        print("Lower\n")
        continue
    elif n < 25:
        print("Higher\n")
        continue
    
    print("Good")
    break